projects/collections/docker-compose.yml

```diff
@@ -49,14 +49,3 @@ services:
       PLEK_SERVICE_STATIC_URI: assets.publishing.service.gov.uk
       VIRTUAL_HOST: collections.dev.gov.uk
       BINDING: 0.0.0.0
-
-  collections-app-account:
```

Comment:
I know it's pretty minor, but wouldn't `account-api` be needed for the logged in header?
I see we've just removed this here,

So presumably the thing would be to have two separate terminals one with:
`govuk-docker up finder-frontend-app` and the other with `govuk-docker up collections-app` if you needed the full brexit checker journey?

Reply:
No, [collections doesn't use account-api at all](https://github.com/alphagov/collections/blob/main/app/controllers/concerns/account_concern.rb)

Reply:
great! ok!